### ISSUE TITLE
Need any help with this?

### ISSUE THREAD

**Issue by JamesDBartlett3** [Hugoberry/pbix-diff-action#2]
Hi there!

This looks like a great way to document Power BI reports in version control. Bravo on the work you've done so far!

Are you still actively developing this solution, and if so, could you use any help? 
I'd be particularly interested in extending it to work with Azure DevOps pipelines. 

Cheers!

- James

**Comment by Hugoberry** [comment]
Sorry got carried away with other "work". Let me see if I can resurrect this project in a few days.

**Comment by JamesDBartlett3** [comment]
Actually, this may not be necessary anymore, since [a full source control solution for PBIX files](https://github.com/action-bi-toolkit/pbi-tools) was just released earlier this week.

**Comment by Hugoberry** [comment]
Awesome! Will check it out.
Oh wait, this is not some official library. I'm sorry but in my experience, this can become a blood bath once MS decides to change their file format. That happening about once a year.

**Comment by JamesDBartlett3** [comment]
True, but the same thing will happen to this project if Microsoft changes the PBIX format, right?
